README.md

```diff
@@ -1,4 +1,4 @@
-# Async.js
+![Async Logo](https://raw.githubusercontent.com/caolan/async/master/logo/async-logo_readme.jpg)
```

Comment:
http://stackoverflow.com/a/36313391/1517919
But sounds good, I'd rather gaurentee it works on npm than fiddle with npm ignore stuff